doc/API/centreon-api-v2.yaml

```diff
@@ -296,6 +302,56 @@ paths:
               application/json:
                 schema:
                   $ref: '#/components/schemas/ErrorApiResponse'
+  /configuration/proxy:
+    get:
+      tags:
+        - Proxy
+      summary: Display the default configuration of the Centreon proxy
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Configuration.Proxy'
+        403:
```

Comment:
should not be 401 ?

Reply:
according to rfc, it is 403 :) https://tools.ietf.org/html/rfc7231#section-6.5.3